### ISSUE TITLE
Implement GUI for user-friendly configuration

### ISSUE THREAD

**Issue by Skryvvara** [Skryvvara/FocusFrame#4]
# What is the desired functionality?

The application should provide a GUI so users can configure the software without editing the config file manually.

# What is the current behavior?

There's no GUI

# What would be an ideal solution?

Ideally a library that works on windows, mac, linux and bsd should be used. This pretty much goes hand in hand / depends on #1

**Comment by Skryvvara** [comment]
First version of the GUI is merged into main, not perfect currently but I'm happy enough with it as of now
